eng/ApiCompatability/PreviousReleaseApiCompat.props

```
@@ -0,0 +1,13 @@
+<Project>
+  <PropertyGroup>
+    <RunPreviousReleaseApiCompat>true</RunPreviousReleaseApiCompat>
+    <!-- only validate that the current implementation is compatible
+         with the old one, not vice-versa (since latest may have new
+         API missing from old) -->
+    <RunPreviousReleaseApiCompatForSrc>true</RunPreviousReleaseApiCompatForSrc>
+    <RunPreviousReleaseMatchingRefApiCompat>false</RunPreviousReleaseMatchingRefApiCompat>
+    <PreviousReleaseAPICompatLeftOperand>".NET Core 3.0 reference"</PreviousReleaseAPICompatLeftOperand>
```

In other places we use "implementation"
```suggestion
    <PreviousReleaseAPICompatLeftOperand>".NET Core 3.0 implementation"</PreviousReleaseAPICompatLeftOperand>
```

or we need to change in other places to use "reference" for the `PreviousReleaseAPICompatLeftOperand`

---

Similar to above try to use more generic names so you don't need to version these.